### PR TITLE
Update gotomeeting to 8.5.0.6956

### PR DESCRIPTION
**Pull request by Saklad5**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by Saklad5** [comment]
If anyone can figure out a cleaner way of installing this, I’m all for it. It was previously using a version-specific URL that was nevertheless labelled `latest`, which is obviously not ideal. The download URL seems pretty much arbitrary, and the process of downloading it normally is… bizarre to say the least.

**Comment by adidalal** [comment]
I found http://support.citrixonline.com/en_US/Meeting/all_files/G2M010014 - that gives me a link of http://support.citrixonline.com/en_US/Meeting/downloaddocument/G2MD00250 for this latest version.

File: Casks/gotomeeting.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'gotomeeting' do
-  version :latest
-  sha256 :no_check
+  version '8.5.0.6956'
+  sha256 '5722fb63a00c320a20d071f8cb00d2fe93a1c4f73cbf8b80602021a8f8fb82de'
 
   # support.citrixonline.com was verified as official when first introduced to the cask
-  url 'https://support.citrixonline.com/servlet/fileField?entityId=ka3380000000FRYAA2&field=Content__Body__s'
+  url 'https://support.citrixonline.com/servlet/fileField?retURL=%2Fapex%2FCPDownloadStarter%3FarticleLinkId%3DG2MD00250%26l%3Den_US%26product%3DMeeting&entityId=ka338000000CjPXAA0&field=Content__Body__s'
```
Comment (adidalal) [inline Casks/gotomeeting.rb]: Let's update this to the cleaner URL, incorporating the `after_comma`.

**Comment by Saklad5** [comment]
@adidalal That’s not a download link, unfortunately, just a redirect to a similarly messy one.

File: Casks/gotomeeting.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 cask 'gotomeeting' do
-  version :latest
-  sha256 :no_check
+  version '8.5.0.6956'
```
Comment (adidalal) [inline Casks/gotomeeting.rb]: might make sense to have `version '8.5.0.6956,G2MD00250'` so the download URL is linked to the version.

**Comment by reitermarkus** [comment]
Probably only the `ka338000000CjPXAA0` part needs to change.

**Comment by Saklad5** [comment]
@reitermarkus I just tried removing that parameter. Unfortunately, `curl` still downloads a useless webpage without it.

**Comment by commitay** [comment]
> Probably only the ka338000000CjPXAA0 part needs to change

@Saklad5 This works. Unfortunately https://support.citrixonline.com/en_US/Meeting/all_files/G2M010014 is not stable, so it can't be used as an `appcast`.

```
cask 'gotomeeting' do
  version '8.5.0.6956,ka338000000CjPXAA0'
  sha256 '5722fb63a00c320a20d071f8cb00d2fe93a1c4f73cbf8b80602021a8f8fb82de'

  # support.citrixonline.com was verified as official when first introduced to the cask
  url "https://support.citrixonline.com/servlet/fileField?entityId=#{version.after_comma}&field=Content__Body__s"
  name 'GoToMeeting'
  homepage 'https://www.gotomeeting.com/'

  app 'GoToMeeting.app'
end

```

**Pull request merged**